package.json

```diff
@@ -126,7 +126,7 @@
     "govuk_frontend_toolkit": "^9.0.0",
     "hammerjs": "^2.0.8",
     "helmet": "^3.21.3",
-    "http-proxy-middleware": "1.1.x",
+    "http-proxy-middleware": "^3.0.5",
```

Comment:
There are some incompatibilities with this, so careful testing will be required. From memory we needed to do a fair bit to get this working in MC.